src/locale/locales/de/messages.po

```diff
@@ -97,35 +79,35 @@ msgstr "Konto"
 
 #: src/view/com/profile/ProfileHeader.tsx:245
 msgid "Account blocked"
-msgstr ""
+msgstr "Konto blockiert"
 
 #: src/view/com/profile/ProfileHeader.tsx:212
 msgid "Account muted"
-msgstr ""
+msgstr "Konto stummgeschaltet"
 
 #: src/view/com/modals/ModerationDetails.tsx:86
 msgid "Account Muted"
-msgstr ""
+msgstr "Konto stummgeschaltet"
```

Comment:
Should we follow the casing here?

```suggestion
msgstr "Konto Stummgeschaltet"
```

Reply:
I wasn't sure what the convention in German on capitalisation in the circumstance of user interface text like this was, so I had just left them in sentence case. But I will follow whatever German speakers like yourself think is best of course :)

Reply:
I'm a fan of that personally, but maybe others can have a look as well. It's not universal 😅